index.html

```diff
@@ -100,196 +107,329 @@ <h2 id="connect">Let's Connect</h2>
         </li>
       </ul>
     </section>
+    <!-- WORKS SECTION -->
     <section id="works-container">
       <ul id="works">
         <li class="work-card">
-          <img
-            src="images/work_1.jpg"
-            alt="work_tonic_daily_personalized_reads"
-          />
-          <h2>Tonic</h2>
-          <ul class="experience">
-            <li>Canopy</li>
-            <li>&#x2022;</li>
-            <li>Back End Dev</li>
-            <li>&#x2022;</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="technologies">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
+          <ul class="work-card__ul">
+            <li class="work-card__header">
+              <picture>
+                <source
+                  srcset="images/work_1_lg.png"
+                  media="(min-width: 768px)"
+                />
+                <img
+                  src="images/work_1.jpg"
+                  alt="work_tonic_daily_personalized_reads"
+                />
+              </picture>
+            </li>
+            <li class="work-card__body">
+              <ul>
+                <li class="work-card__title">Tonic</li>
+                <li>
+                  <ul class="work-card__experience">
+                    <li>Canopy</li>
+                    <li>&#x2022;</li>
+                    <li>Back End Dev</li>
+                    <li>&#x2022;</li>
```

Comment:
- [ ] Kindly for semantic tags, instead of using `ul` for wrapping your items inside use `section` or `article` depending on the information that you are using inside. For example, `ul` tags are for listing items, like following a recipe. But in this list of cards, you have articles for example. There's no list of items, so `ul` and `li` are not semantic in this section. Kindly replace it with the corresponding one. [section ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)and [article ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)